### ISSUE TITLE
Conduits seepage and xsection_area

### ISSUE THREAD

**Issue by tscheuch** [OpenWaterAnalytics/pyswmm#247]
Hi everyone! 
I am trying to get the seepage loss on conduits. When I use the .seepage_rate I get the following error:

![image](https://user-images.githubusercontent.com/61126547/87440413-f18a0080-c5bf-11ea-8974-b227691f3ed5.png)

Do you know what happens? 
Also, is this seepage_rate the user supplied one or the real seepage loss as a function of the width of the water surface?

Trying to calculate the seepage time series by my self in every time step following the methodology exposed on the Hydraulics SWMM manual, I realized that the .ups_xsection_area and ds_xsection_area are returning zeros when the conduit depth is not.

![image](https://user-images.githubusercontent.com/61126547/87441640-5003ae80-c5c1-11ea-9333-dc28e3014998.png)

Thank you again and congratulations for the work with the library!

**Comment by bemcdonnell** [comment]
@tscheuch,

this is super interesting that this property is provided in pyswmm. looking at the SWMM function that PySWMM calls, i don't even see seepage rate therein:

https://github.com/OpenWaterAnalytics/Stormwater-Management-Model/blob/419e406abc19173fc42be40d2cfccd3348f4c641/src/toolkitAPI.c#L615-L657

So the error pyswmm is throwing makes sense but the fact that we opened up this feature in the entry point has me a little bit puzzled.  @katmratliff and @jennwuu, do either of you remember when we added this? (it was probably me... haha)

**Comment by katmratliff** [comment]
@bemcdonnell it looks like the last time the seepage rate getter/setter was touched was 3 and 4 years ago, so before my time :)

**Comment by jennwuu** [comment]
I think it was before my time as well. But it would be great feature to add. :o

**Comment by katmratliff** [comment]
agreed, it looks pretty straightforward to do so on the SWMM API side.

**Comment by bemcdonnell** [comment]
Yup! I'm totally responsible for this bug. @tscheuch, would you be open to helping us fix this?

**Comment by tscheuch** [comment]
For sure!
I have no idea how pyswmm interacts with SWMM codes. The thing is that the seepage rate of a conduit is calculated with a user supplied rate. I imagine that getting/setting this parameter shouldn't be difficult.
It would be interesting to report the real seepage loss of the conduit in every step (that is calculated on link.c and depends on the the width of the water surface). The losses time series aren't reported by SWMM, only in a aggregated way on the .rpt file.

**Comment by dickinsonre** [comment]
Some 3rd party software do report the time step seepage for each link and it is often useful.
